src/Styleguide/Elements/Message.tsx

```diff
@@ -0,0 +1,22 @@
+import { color } from "@artsy/palette"
+import { Sans } from "@artsy/palette"
+import React from "react"
+import { Flex, FlexProps } from "Styleguide/Elements/Flex"
+
+export const StyledFlex = Flex.extend`
+  background-color: ${color("black5")};
```

Comment:
this should also have `border-radius: 2px`

Reply:
Good catch! 2px is so subtle it didn't even register with me. I still struggle to see it now unless I zoom in :D